backend/src/routes/v2/model/getModel.ts

```diff
@@ -0,0 +1,36 @@
+import bodyParser from 'body-parser'
+import { Request, Response } from 'express'
+import { z } from 'zod'
+
+import { parse } from '../../../middleware/validate.js'
+import { ModelInterface } from '../../../models/v2/ModelModel.js'
+import { getModelById } from '../../../services/v2/model.js'
+
+export const getModelSchema = z.object({
+  params: z.object({
```

Comment:
When importing 'z' could we give it useful name? 'z.object' and the like isn't the nicest to read

Reply:
Whilst I'm generally a proponent of using useful names for things, this is how it's exported by default and also how it's used in the official documentation, so I think it's acceptable to keep it as 'z' (although I'm happy to have it changed to 'zod' if I'm overruled by the majority).

Reply:
I'm happy to yield on this one- might just be a nice to have for me personally.